db/migrate/20220503071925_add_created_by_to_message.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedByToMessage < ActiveRecord::Migration[6.1]
+  def change
+    add_column :messages, :created_by, :string, default: nil
```

Comment:
let's use campaign_id as suggested above for the attribute so that it stays consistent with the attribute we have in conversation. We will also be able to add belongs to => campaign association easily then.

Reply:
Do we need it as a separate attribute or can we add it in the content_attributes? For the large part, this field would be nil, right?

Reply:
We can have it in content_attributes 👍

Reply:
@pranavrajs At the moment the content_attributes column is JSON type. So it's difficult to query campaign_id from content_attributes. We need to take this as tech debt.

Reply:
@aswindevps I don't think we are querying campaign_id anywhere in this change, right? I guess what @sojan-official meant was that If at all required, we could query it from content_attributes.

Reply:
After this change, we will have to figure out the number of outgoing messages without campaign_id. 
ref: https://github.com/chatwoot/chatwoot/blob/4f9d41962a3f98206297679ff65d72b9ab3af214/app/models/message.rb#L171

Reply:
`if outgoing? && conversation.messages.outgoing.where("(additional_attributes->'campaign_id') is null").count == 1`